proc_scripts/align_slice.py

```diff
@@ -1,4 +1,5 @@
 from pyspecdata import *
+import numpy as np
```

Comment:
question: why don't we use the correlate function of either scipy or numpy?

Reply:
because then you would need to figure out the axis

Reply:
so don't we adjust the axis anyhow? why would this be a problem?

Reply:
If we do this with pyspecdata, the offset between the peaks is given as a frequency in Hz, rather than an index number we would have to convert. Also, there is no sense to using one of those functions, since they would only replace some 2 lines of code.